client/payment-request/index.js

```diff
@@ -180,16 +184,25 @@ jQuery( ( $ ) => {
 		 */
 		startPaymentRequest: ( options ) => {
 			const paymentRequest = getPaymentRequest( options );
-
 			const elements = api.getStripe().elements();
 			const prButton = wcpayPaymentRequest.createPaymentRequestButton(
 				elements,
 				paymentRequest
 			);
 
+			const doActionPaymentRequestAvailability = ( args ) => {
+				defaultHooks.doAction(
```

Comment:
`defaultHooks` didn’t seem to work for me initially when I had WordPress 5.7 installed. It worked perfectly after updating to 5.8 though.

Reply:
Thank you for the review and https tip 👍 💯 .

I only tested it on WP 5.8. I'll test it on 5.7 and 5.6.

Reply:
@adrianduffell  I tested it locally with 5.7 and it looks like there must be something not working with ngrok configuration.

Does your `/cart` endpoint work without an error?

I think the problem is that `/cart` endpoint fails and the action never gets fired.

I'm gonna see if I can get around it locally. If this is the case, I think this is unrelated to the code.

![Screen Shot 2021-08-05 at 4 17 37 PM](https://user-images.githubusercontent.com/4723145/128433486-09d20d74-0b4e-4ac4-8ef4-41d16909715f.jpg)

Reply:
@adrianduffell  I was able to make it work with WP 5.7 in an incognito session. I think WooCommerce tries to load a file from HTTP when you're signed in as an admin, which causes "blocked/mixed-content" error in Chrome. Could you try performing the test in an incognito session, please?

Reply:
Thanks @moon0326. Hmm 🤔 I don't get the same mixed-content issue locally. My `/cart` seems to work fine and isn't doing the same http location redirect to http.

<img width="1201" alt="Screen Shot 2021-08-06 at 12 29 57 pm" src="https://user-images.githubusercontent.com/9312929/128456226-cab2e028-7145-4271-81bf-13ffd6d6aba4.png">

Unfortunately I'm still seeing the issue when using Chrome Incognito, admin signed out, WP 5.7 😅  

This is the error I see on most shop pages (product, cart, checkout).

<img width="1054" alt="Screen Shot 2021-08-06 at 12 31 27 pm" src="https://user-images.githubusercontent.com/9312929/128456306-02df523e-dffa-4910-851d-9687247833e4.png">

I can also reproduce the error using http via `http://localhost:8082/cart/` with WP 5.7. How does localhost cart with 5.7 work for you?

Reply:
Interesting 🤔 Well, it stopped working for me even in an incognito session. I still get `blocked/mixed-contetn` error. Clicking the update cart button twice works for me though.

Yeah, it definitely looks like it's not finding the method. I'll clean install WP and test it again.

Reply:
@adrianduffell I was able to reproduce it on 5.7. 

I don't know exactly how `defaultHooks` become null, but using `doAction` directly worked on both 5.7 and 5.8.

Updated in https://github.com/Automattic/woocommerce-payments/pull/2610/commits/cb48c0dc31be1a7d711217e4c99a99b8f29c6858